package.json

```diff
@@ -9,6 +9,9 @@
     "image",
     "raster"
   ],
+  "alias": {
+    "txml/txml": "txml/dist/txml"
```

Comment:
Parcel `v1` cannot resolve package exports used by `txml` (https://github.com/TobiasNickel/tXml/pull/23). This alias makes parcel v1 happy, but keeps the modern import name in the source for other bundlers.